broadinstitute/single_cell_portal_core
# Collections UX - the return (SCP-3595)

SCP-3595  The collections UX has now been greenlit for merge (although it should not be released without SCP-3685).  This reverts the revert commits, and adds some last-minute UX polish from Jerome.  Changes are removing the icon on the Browse Collections button, left-aligning the 'Collections' header, and adding a 'Home' link to that header.  

![image](https://user-images.githubusercontent.com/2800795/134056645-7dc9c949-d563-42bb-a339-dda15c16fda9.png)

![image](https://user-images.githubusercontent.com/2800795/134056811-e6fcb7fc-2fc0-43d2-9111-2417bdcb8d75.png)

TO TEST:
1. Go to home page, make sure 'Browse Collections' button appears
2. Click 'Browse Collections', view the list page, and confirm the styling appears correct
